blocks/footer/footer.css

```diff
@@ -1,14 +1,63 @@
 footer {
-  padding: 2rem;
-  background-color: var(--overlay-background-color);
-  font-size: var(--body-font-size-s);
+  --footer-text-color: #fff;
+
+  color: var(--footer-text-color);
+  background-color: #000;
```

Comment:
could be moved to styles.css and used here as variable name

Reply:
done in https://github.com/hlxsites/macquarie-help-centre/pull/14/commits/9063d8792c25fa39bc9ac98b7e2e547237ccbe05